app/marketing/migrations/0018_add_upcoming_date_tag.py

```diff
@@ -0,0 +1,16 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('marketing', '0017_auto_20200903_1904'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='UpcomingDate',
+            name='context_tag',
+            field=models.TextField(default='', max_length=255),
```

Comment:
this migration need to be removed and recreated since the model now can be blank